packages/gestalt/src/ModuleTitle.js

```diff
@@ -30,17 +31,29 @@ export default function ModuleTitle({
     },
   };
 
-  const { color, icon: iconName } = MODULE_TYPE_ATTRIBUTES[type];
+  const { color, icon: iconName } = TYPE_ICON_ATTRIBUTES[type];
 
   return (
     <Flex gap={2}>
       {iconName && (
         <Icon accessibilityLabel={iconAccessibilityLabel ?? ''} color={color} icon={iconName} />
       )}
 
-      <Text color={color} truncate weight="bold">
-        {title}
-      </Text>
+      <Flex.Item minWidth={0}>
+        <Text color={color} truncate weight="bold">
+          {title}
+        </Text>
+      </Flex.Item>
+
+      {badgeText && (
+        <Box
+          dangerouslySetInlineStyle={{ __style: { top: '1px' } }}
```

Comment:
nudged a hair for visual centering

Reply:
If you remove relative, do you still need the top px? I played with the css in the Docs and seemed to work, but might be wrong.

Reply:
Yep, the `position` is necessary for the `top` to take effect. And the `top` is needed to nudge the badge down a hair so it's visually vertically centered (as contrasted with what flex thinks vertical centering is)

Reply:
could the badge be centered on Flex? alignItems="center"? Just curious.

Reply:
That's what I mean by "visually centered" as opposed to just "centered". This is a common occurrence when aligning text and other visual elements: what flex thinks is "centered" is actually off by a pixel or two.